contracts/test/CompoundGovernor.t.sol

```diff
@@ -113,6 +124,24 @@ contract Propose is CompoundGovernorTest {
         _submitProposal(_proposer, _proposal);
     }
 
+    function testFuzz_RevertIf_ProposerHasActiveProposal(uint256 _elapsedTime) public {
```

Comment:
Shouldn't this be `_elapsedBlocks` like the `testFuzz_ProposerCanSubmitAnotherProposalAfterVotingPeriod`